### PR TITLE
issue #16 - add label box / label module

### PR DESCRIPTION
**Pull request by magsout**
[WIP]

@zoepage if you could tell me, my approach is good for you and respect the semantics that you have in your mind. fell free to push or commit on this branch.

is still wip.

### CODE REVIEW

**Comment by zoepage** [comment]
@magsout could you take a look at the merge conflicts?

**Comment by magsout** [comment]
need to fix again.

**Comment by zoepage** [comment]
@magsout  could you rebase please? There 1-2 bugs that should be fixed by it. :)

The way looks exactly what we need, so you're on the right path. Happy to do a proper review when you are done? If you'd like detailed feedback right now, happy to provide?

**Comment by magsout** [comment]
done @zoepage

**Comment by zoepage** [comment]
@magsout do you plan to add the label selection popup? :) 
https://github.com/webcompat/webcompat.com/issues/1522#issuecomment-297246594

**Comment by magsout** [comment]
@zoepage yes, I pushed my work just to be sure what what I did was ok ;) I'm going to continue my work ;

**Comment by zoepage** [comment]
\o/

File: docs/css/main.css [hunk 1]
```diff
@@ -46,6 +47,15 @@
   }
 }
 
+<<<<<<< HEAD
+=======
```
Comment (miketaylr) [inline docs/css/main.css]: oops `<<<<<<< HEAD` :)

**Comment by magsout** [comment]
@zoepage I pushed my last commit.

2 things: 
- what about cog icon ? What I can use for this button ?
- About Header'sbackground-color, what is the best to handle thoses colors ?
- issue lists ?

File: docs/css/main.css (hunk 1 above)
Comment (magsout) [reply docs/css/main.css]: yeah yeah ;) I messed my rebase, I guess

**Comment by zoepage** [comment]
@magsout 

> what about cog icon ? What I can use for this button ?

https://github.com/webcompat/design/blob/master/docs/svg/icons/svg-settings.svg It is included in the sprite.

> About Header's background-color, what is the best to handle thoses colors ?

It depends... If you just look for colors, https://github.com/webcompat/design/blob/master/docs/css/src/variables.css#L30-L42 should work or did I missunderstand?

> issue lists ?

Is this related to your question before or another question? :)

**Pull request merged**